src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/CognitiveServicesAccount/GetAzureCognitiveServicesAccountSkus.cs

```diff
@@ -15,45 +15,90 @@
 using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
 using Microsoft.Azure.Management.CognitiveServices;
 using Microsoft.Azure.Management.CognitiveServices.Models;
+using Microsoft.Azure.Commands.Management.CognitiveServices.Models;
+using System;
+using System.Linq;
 using System.Management.Automation;
-using CognitiveServicesModels = Microsoft.Azure.Commands.Management.CognitiveServices.Models;
 
 namespace Microsoft.Azure.Commands.Management.CognitiveServices
 {
     /// <summary>
     /// Get Available Skus for Cognitive Services Account
     /// </summary>
-    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "CognitiveServicesAccountSkus"), OutputType(typeof(CognitiveServicesModels.PSCognitiveServicesSkus))]
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "CognitiveServicesAccountSkus", DefaultParameterSetName = GetSkusWithAccountParamSetName),
+        OutputType(typeof(PSCognitiveServicesSkus), typeof(ResourceSku))]
     public class GetAzureCognitiveServicesAccountSkusCommand : CognitiveServicesAccountBaseCmdlet
     {
+        protected const string GetSkusWithAccountParamSetName = "GetSkusWithAccount";
+        protected const string GetSkusWithFilterParamSetName = "GetSkusWithFilter";
+
+        [Parameter(
+            ParameterSetName = GetSkusWithFilterParamSetName,
+            Position = 0,
```

Comment:
@yangyuan for the `-Type` and `-Location` parameters, please remove the `Position` property since the parameters are optional

Reply:
cool, thanks.